fastlane/Fastfile

```diff
@@ -217,6 +217,10 @@ GH_ORG_NAME="woocommerce"
   lane :finalize_release do | options |
     ios_finalize_prechecks(options)
     unless ios_current_branch_is_hotfix
+      check_translation_progress(
+        glotpress_url: 'https://translate.wordpress.com/projects/woocommerce/woocommerce-ios/',
+        abort_on_violations: false
+      )
```

Comment:
Same feedback as https://github.com/wordpress-mobile/WordPress-Android/pull/14828#discussion_r667118977: I think we also need to check release notes.